src/templates/utils-common.js

```diff
@@ -24,8 +24,14 @@ export const matchBrowserLocale = (appLocales, browserLocales) => {
 
   // First pass: match exact locale.
   for (const [index, browserCode] of browserLocales.entries()) {
-    if (appLocales.includes(browserCode)) {
-      matchedLocales.push({ code: browserCode, score: 1 - index / browserLocales.length })
+    if (appLocales.forEach(appLocale => {
+      if (appLocale.toLowerCase().includes(browserCode.toLowerCase())) {
+        matchedLocales.push({ code: browserCode, score: 1 - index / browserLocales.length })
+        return true
+      } else {
+        continue
+      }
```

Comment:
You need to push app locale since browser locale case might not match and then that will not match in the calling code.

Also here is suggested cleaner code that doesn't use functions (forEach):
```suggestion
    for (const appCode of appLocales) {
      if (appCode.toLowerCase() === browserCode.toLowerCase()) {
        matchedLocales.push({ code: appCode, score: 1 - index / browserLocales.length })
        break;
      }
```

Reply:
Also please add a test for comparing locales in case-insensitive manner (in `unit.test.js`).

Reply:
Thank you! I much appreciate your helping.